lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -393,6 +393,13 @@ def test_create_ccx(self, ccx_name='New CCX'):
         ccx = CustomCourseForEdX.objects.get()
         course_enrollments = get_override_for_ccx(ccx, self.course, 'max_student_enrollments_allowed')
         self.assertEqual(course_enrollments, settings.CCX_MAX_STUDENTS_ALLOWED)
+        # check if the course display name is properly set
+        course_display_name = get_override_for_ccx(ccx, self.course, 'display_name')
+        self.assertEqual(course_display_name, ccx_name)
+
+        # check if the course display name is properly set in modulestore
+        course_display_name = self.mstore.get_course(ccx.locator).display_name
+        self.assertEqual(course_display_name, ccx_name)
```

Comment:
@taranjeet From a high level I'm not sure why this check would be failing. Did you test these steps manually (by doing what you did [earlier](https://github.com/edx/edx-platform/pull/18808#discussion_r215274883), replacing relevant parts with [this code](https://github.com/edx/edx-platform/pull/18808#discussion_r215274883)), to see if that would result in the expected behavior?

CC @ormsbee

Reply:
@taranjeet: Yeah, it's not clear to me how that test is failing if the feature itself is working. If you've gone step by step through it in a debugger, can you describe where it's going wrong? Not finding the override? Not using the override? Grabbing a cached version from somewhere in some way?

@itsjeyd: I think I'm good with the PR once the test passes. Please let me know if you need me for more review. Otherwise, I'll wait until the test passes and you give a thumbs up, and then merge.

Reply:
@ormsbee @itsjeyd : I have figured out what was missing. We need to override some settings so that the modulestore can work properly. The code is updated now.